apps/namadillo/src/atoms/balance/atoms.ts

```diff
@@ -28,37 +27,17 @@ export type TokenBalance = AddressWithAsset & {
   dollar?: BigNumber;
 };
 
-const DEPRECATED_getViewingKey = async (): Promise<string | undefined> => {
```

Comment:
I explicitly took this out since that interface is no longer valid (no `owner` exists in the extension). Are you saying it should be kept?

Reply:
ah, I see the new conflicts. I promise I won't merge with conflicts :D

Reply:
@jurevans unfortunately we still need to preserve this. Because we could have someone that do not updated the extension, but is already using the new Namadillo, then we still need to provide some support for them

Reply:
@euharrison I was thinking about this, and to keep that function, we'll also need to keep `owner` in Extension, which means I need to revert some things in extension, right? As soon as we publish the Chrome extension, it should auto-upgrade on users machines, so we can assume that they'll be on the latest, and I plan to announce this to users on Discord who may be using the test build I provided.

Reply:
@euharrison just FYI, there are some other breaking changes if there is a mismatch between this Namadillo & extension, so if our plan is to support previous and current version, there will be quite a bit of handling needed, and I personally think it would be best to move everything to the latest, inform people to update their extension (it should be handled automatically for Chrome Store users), and ensure we get Namadillo published on Dry-Run & Housefire. Do you think that would be too problematic?